sklearn/externals/joblib/externals/cloudpickle/cloudpickle.py

```diff
@@ -933,7 +933,7 @@ def subimport(name):
 
 
 def dynamic_subimport(name, vars):
-    mod = imp.new_module(name)
+    mod = types.ModuleType(name)
```

Comment:
```suggestion
    mod = importlib.util.module_from_spec(name)
```

Reply:
I followed [the documentation](https://docs.python.org/3.5/library/imp.html#imp.new_module) as well..
Anyway, it looks better to use `importlib`. Thanks!

Reply:
I had to change import imp to import importlib for it to work. Thanks, anyway!